src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -120,11 +133,31 @@ const CCPDashboardPage = ({ match }: RouteComponentProps<TParams>) => {
     [patients]
   );
 
-  if (loading) {
+  if (loading || ccpLoading) {
     return <LoadingState />;
   }
+
+  const currentCcp = ccpInfo.collectionPoint;
+
+  const menuBarTitle = (
+    <>
+      {currentCcp.name}
+      <div className={classes.menuBarTitle}>
+        <LocationOnOutlinedIcon className={classes.locationIcon} />
+        <Typography variant="caption" className={classes.menuBarTitle}>
+          100 University
```

Comment:
hard-coded for now; @petorsong hope this is ok

Reply:
yep - any location/address related item for CCPs will be covered under @aaronabraham311 's maps/location work